### PR TITLE
Mds 3946 add notes field to TSF in core and minespace

### PR DESCRIPTION
**Pull request by dragancolic-bcgov**
# Main

- What did you focus on in this PR?
- Keep all headers and just leave a single list item 'N/A' if there's no content for that header, otherwise remove N/A and add your details!

# Other

- What other side fixes, tweaks did you add on the side?
- N/A

# How to test

- N/A

# Notes

- Any notes/screenshots for this PR?
- N/A

### CODE REVIEW

File: services/core-api/app/api/mines/tailings/resources/tailings.py [hunk 1]
```diff
@@ -53,6 +53,12 @@ class MineTailingsStorageFacilityResource(Resource, UserMixin):
         help='GUID of the party that is the Engineer of Record for this TSF.',
         location='json',
         store_missing=False)
+    parser.add_argument(
+        'notes',
```
Comment (JustinMacaulay) [inline services/core-api/app/api/mines/tailings/resources/tailings.py:57]: Does this not need to be added to the put?

File: services/core-web/src/components/mine/Tailings/MineTailingsTable.js [hunk 2]
```diff
@@ -7,10 +7,10 @@ import {
   getITRBExemptionStatusCodeOptionsHash,
 } from "@common/selectors/staticContentSelectors";
 import * as Strings from "@common/constants/strings";
+import { Button } from "antd";
```
Comment (JustinMacaulay) [inline services/core-web/src/components/mine/Tailings/MineTailingsTable.js:10]: I dont see any changes involving this button is it needed?

**Review by JustinMacaulay — COMMENTED** [review]
Just a few comments.

File: services/core-web/src/components/mine/Tailings/MineTailingsTable.js (hunk 2 above)
Comment (dragancolic-bcgov) [reply services/core-web/src/components/mine/Tailings/MineTailingsTable.js:10]: Thanks for the review. Oh this one was at line 13. So yes it's being required for the lines 95 and 104.

File: services/core-api/app/api/mines/tailings/resources/tailings.py (hunk 1 above)
Comment (dragancolic-bcgov) [reply services/core-api/app/api/mines/tailings/resources/tailings.py:57]: Thanks for the review. Yup, this one is required for modal create/update.

**Review by areyeslo — APPROVED** [review]
Great job Dragan!! Your implementation works like a charm.

**Pull request merged**